turtle2.py

```diff
@@ -0,0 +1,130 @@
+#submitted by Keren Or Hadad 208025205
+import string
+
+
+class Message:
+
+    def _init_(self, m_id, body, sender):
+        self.message_id = m_id
+        self.message_body = body
+        self.message_sender = sender
+        self.message_read = False
+
+    def get_details(self):
+        """
+        The function return details of a message.
+        Returns: String with message's details.
+        """
+        return {"Id: " + str(self.message_id) + ", Body: " + self.message_body + ", Sender: " + self.message_sender +\
+                 ", Read: " + str(self.message_read)}
+
+    def set_read(self, status_read):
+        """
+        The function that change status of message. Read or not read.
+        Parameters
+        -------
+        status_read : bool
+            New status of read.
+        """
+        self.message_read = status_read
```

Comment:
you were asked to implement str and len methods, please fix this.
https://www.tutorialsteacher.com/python/magic-methods-in-python